src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball firstBall = Lottery.getRandomBall();
+        System.out.println(firstBall.toString());
+        Ball secondBall = Lottery.getRandomBall();
+        System.out.println(secondBall.toString());
+        Ball thirdBall = Lottery.getRandomBall();
+        System.out.println(thirdBall.toString());
```

Comment:
As with `firstBall` and `secondBall`, you can pass `thirdBall` directly to `println`.